### PR TITLE
[#153213089] Update landing page content

### PR DESCRIPTION
**Pull request by 46bit**
## What

Update the post-login landing page with new content designed by the product team. See commit messages for details.

## How to review

View the `/next-steps` page and give feedback on the new content. Of particular interest, was removing the mention of "targets" the right call (see the 3rd commit)?

## Who can review

Not @46bit @dcarley

### CODE REVIEW

File: views/next-steps.erb [hunk 1]
```diff
@@ -19,32 +19,28 @@
...
     <div class="column-two-thirds">
       <h1 class="heading-large">
-        Start using GOV.UK PaaS
+        Access GOV.UK PaaS
       </h1>
 
       <p class="lede">
-        A lot of the tasks you’ll need to do to operate GOV.UK PaaS will be done from your computers
-        command line interface.
+        Installing the Cloud Foundry command line will let you operate GOV.UK PaaS from your computer
```
Comment (samcrang) [inline views/next-steps.erb]: Do we need a full-stop at the end of this sentence?

**Review by samcrang — CHANGES_REQUESTED** [review]
Looks good, except for my one question. I think keeping mentions of "targets" out makes sense and if it makes the layout better then 👍 .

File: views/next-steps.erb (hunk 1 above)
Comment (46bit) [reply views/next-steps.erb]: I checked with Jess--yup!

Comment (samcrang) [reply views/next-steps.erb]: Nice. I'll merge this tomorrow morning.

**Review by samcrang — APPROVED** [review]
Looks good.

**Pull request merged**